src/app/shared/components/type-of-help/type-of-help.component.html

```diff
@@ -0,0 +1,10 @@
+<div *ngIf="unfoldOnSelect" class="top-unfold" [style.visibility]="topBottomVisibility"></div>
+<mat-card (click)="handleClick()">
+  <div class="icon-wrapper">
+    <mat-icon class="material-icons-outlined">{{ icon }}</mat-icon>
+  </div>
+  <div class="name-wrapper">
+    <strong [innerHTML]="name | translate"></strong>
```

Comment:
Isn't this a potential XSS risk? Alternatives that come to mind:
* `<strong>{{name | translate}}</strong>` 
* `name | translate | safeHtml` https://stackoverflow.com/a/39630507
* `[textContent]="...."`

Reply:
It's safe. Angular sanitize contet passed via [innerHTML].

Reply:
No it doesn't. I just checked with:
```
html = 'this <b>is</b> html'

<div [innerHTML]="html"></div>
```
and it rendered as 
```
<div _ngcontent-blg-c123="">
this
<b>is</b>
html
</div>
```

Angular only sanitises `{{html}}`

Reply:
Yes, it does. :) It just treats ```<b>``` as safe element.

Try to inject something like this:
```<img scr="runtime.js"/><b class="text-center" data-ele="smth">Noclegi</b> <br>i Mieszkania<script>alert();</script>```

Angular will sanitize all potentially unsafe elements and then inject:
```<img><b class="text-center">Noclegi</b> <br>i Mieszkania```